lib/tbot/config/config.go

```diff
@@ -675,10 +675,14 @@ func ReadConfig(reader io.ReadSeeker, manualMigration bool) (*BotConfig, error)
 type CredentialLifetime struct {
 	TTL             time.Duration `yaml:"credential_ttl,omitempty"`
 	RenewalInterval time.Duration `yaml:"renewal_interval,omitempty"`
+	// Used for services which do not abide by the global TTL limit.
+	ignoreMaximumTTL bool
```

Comment:
This was not obvious at first. Can you please add some examples?

Reply:
6a350961a6